src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/SpringOauthIntegrationTest.kt

```diff
@@ -4,30 +4,41 @@ import com.github.tomakehurst.wiremock.client.WireMock
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
 import org.springframework.test.context.ActiveProfiles
+import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.controllers.DraftAdjudicationResponse
 
 @ActiveProfiles("test")
 class SpringOauthIntegrationTest : IntegrationTestBase() {
 
+  private lateinit var intTestData: IntTestData
+  private lateinit var draftCreationResponseForAdjudication1: DraftAdjudicationResponse
+  private lateinit var draftCreationResponseForAdjudication2: DraftAdjudicationResponse
+
   @BeforeEach
   fun beforeEach() {
-    prisonApiMockServer.stubGetAdjudication()
+    intTestData = IntTestData(webTestClient, jwtAuthHelper, bankHolidayApiMockServer, prisonApiMockServer)
+
+    draftCreationResponseForAdjudication1 = intTestData.startNewAdjudication(IntTestData.ADJUDICATION_1)
```

Comment:
We need to make 2 adjudications ready for reporting so we can check the prison API calls use the correct creds